src/components/progress-indicator-item.tokens.json

```
@@ -0,0 +1,7 @@
+{
+  "of": {
+    "progress-indicator-item": {
+      "not-applicable": {"value": "flex"}
+    }
+  }
+}
```

Sorry I missed it in the previous review, but I noticed only now that the "display" part is missing. I think it would be good to add it, so that it is easier to see from the token which CSS property it's meant to target.
```suggestion
{
  "of": {
    "progress-indicator-item": {
      "not-applicable": {
        "display": {"value": "flex"}
       }
    }
  }
}
```